_locales/en/messages.json

```diff
@@ -1153,6 +1153,22 @@
     "message": "Enable spell check of text entered in message composition box",
     "description": "Description of the media permission description"
   },
+  "shortcutConversationModifierDescription": {
+    "message": "Change modifier for conversation changing shortcut",
```

Comment:
Maybe "Modifier to use with conversation change shortcut"? Or maybe phrase it as a generic modifier key?

Reply:
Thanks for the rephrase, it seems I wasn't inspired the day I wrote that... ^^